upgrades/steps_335.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package upgrades
+
+func stepsFor335() []Step {
```

Comment:
No need for empty steps. Just add the ones needed.

Reply:
Same as above.